diesel/src/pg/types/mod.rs

```diff
@@ -16,4 +16,5 @@ pub mod sql_types {
     #[cfg(feature = "uuid")]
     #[derive(Debug, Clone, Copy, Default)] pub struct Uuid;
     pub type Bytea = ::types::Binary;
+    pub type Bpchar = ::types::VarChar;
```

Comment:
Can you make this `#[doc(hidden)]`? In the long run what we need is a better story for using types that aren't provided by diesel from `infer_schema!` (both for apps, but also for third party crates adding support for things like full text search). However, I'm not fully sure how that's going to be done yet, and I acknowledge that there's no way to work around this in the short term. I'm fine with adding this temporarily, but want to be able to remove it once we have a better story for other types besides "use `table!` instead of `infer_schema!`